src/components/Accordion/Accordion.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <button
+    id="accordion"
+    :class="expanded ? null : 'border-b border-gray-100'"
+    class="cursor-pointer flex justify-between min-w-full mb-6 focus:outline-none focus:ring-1 focus:ring-primary-100 focus:border-transparent"
+    :aria-expanded="expanded"
+    aria-controls="slotContent"
+    role="button"
+    @click="expanded = !expanded"
+  >
+    <h2 class="text-primary-300 font-thin text-lg">
+      <b>{{ title }}</b>
```

Comment:
I'm wondering about `font-thin` and then using `<b>`... why is that? Possibly we could just use an appropiate class for it?